src/jit/importer.cpp

```diff
@@ -11226,6 +11226,23 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 #endif // _TARGET_64BIT_
                 }
 
+#ifdef FEATURE_CORECLR
+                if ((oper == GT_ADD) || (oper == GT_SUB) || (oper == GT_MUL) || (oper == GT_DIV) || (oper == GT_MOD))
```

Comment:
What about the unsigned versions of these operators? Also, do we need to take logical operators into account?